### ISSUE TITLE
Add aria-label attribute to top navigation items

### ISSUE THREAD

**Issue by ltsheu** [18F/dol-whd-14c#306]
Notes from L. Sheu:

/DOL.WHD.Section14c.Web/src/modules/components/mainNavigationControl/mainNavigationControlTemplate.html

To meet accessibility criteria, we need the WIOA link in the top navigation to have an aria-label attribute to spell out "Workforce Innovation and Opportunity Act" for screen readers (currently the screen reader tries to read WIOA as "why-oh-ah").

- [x] Add **role="link"** to each `<li>` in the top navigation:
`<li role="link" ... >WIOA</li>`
- [x] Add the **aria-label** attribute to render like this: 
`<li aria-label="Workforce Innovation and Opportunity Act" role="link" ... >WIOA</li>`

The top nav links are in a repeater, therefore it seems that all the links will get this new aria-label attribute.  For most of the nav links, the value of the aria-label should match the link text, however in WIOA's case the value of aria-label should NOT match the link text, as shown above. 

So ideally the data that goes into the value of the aria-label should be independent of the data that goes into the link text.

**Comment by jmmcnj** [comment]
@binwang89 to review before moving to Reviewed/closed

**Comment by binwang89** [comment]
@ltsheu @jmmcnj I checked the code, it looks like the code are there. 
please see below screenshot. 
![image](https://user-images.githubusercontent.com/25330625/31958524-0e907ff2-b8c0-11e7-9ed9-49ef3797e693.png)
